src/writer/wf_project.py

```
@@ -82,7 +82,11 @@ def write_files(app_path: str, metadata: MetadataDefinition, components: Dict[st
     _write_root_files(wf_directory, components, context)
     _write_component_files(wf_directory, components, context)
     _remove_obsolete_component_files(wf_directory, components)
-    logger.info("Saved. Time elapsed: %.4fs", time.time() - time_start)
+    project_hash = hashlib.md5()
```

Ruff (0.14.8)

Probable use of insecure hash functions in `hashlib`: `md5`

(S324)